base/src/main/java/butter/droid/base/torrent/TorrentHealth.java

```diff
@@ -91,11 +85,4 @@ public int getStringResource() {
         }
     }
 
-    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
unused. and if we need we can use the `ContextCompat`

Reply:
👍